src/core/Microsoft.PowerToys.Settings.UI/Views/KeyboardManagerPage.xaml

```diff
@@ -0,0 +1,244 @@
+                                <TextBlock Grid.Column="1" 
+                                           Style="{StaticResource BodyTextBlockStyle}"
+                                           VerticalAlignment="Center"
+                                           TextAlignment="Center"
+                                           Text="{Binding To}">
+                                </TextBlock>
+                            </Border>
+                        </Grid>
+                    </DataTemplate>
+                </ListView.ItemTemplate>
+            </ListView>
+            
+            <Grid Margin="{StaticResource SmallTopMargin}">
+                <TextBlock Text="Edit Shortcuts"
+                           Style="{StaticResource SubtitleTextBlockStyle}" />
+                <Button Content="Remap" 
+                        Margin="{StaticResource MediumRightMargin}"
+                        Style="{StaticResource AccentButtonStyle}" 
+                        HorizontalAlignment="Right"/>
+            </Grid>
+
+            <TextBlock Text="To redefine shortcuts, press the remap button"
+                       Margin="{StaticResource XSmallTopMargin}"/>
+
+            <StackPanel Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="Shortcuts for:"
+                           FontWeight="Bold">
+                </TextBlock>
+                <TextBlock Text="Config-1"
+                           Margin="{StaticResource XSmallLeftMargin}"
+                           FontWeight="ExtraLight" >
+                </TextBlock>
+            </StackPanel>
+
+            <StackPanel Margin="{StaticResource SmallLeftMargin}"
+                        Orientation="Horizontal">
+                <TextBlock Text="From:"
+                           Width="200">
+                </TextBlock>
+                <TextBlock Text="To:"
+                           Width="200" 
+                           FontWeight="ExtraLight" >
+                </TextBlock>
+            </StackPanel>
+
+            <ListView x:Name="RemapShortcutsList"  
+                      ItemsSource="{StaticResource dummyData}" 
+                      BorderBrush="LightGray" 
+                      BorderThickness="1" 
+                      Width="400"
+                      MaxHeight="150"
+                      Margin="{StaticResource SmallLeftTopRightBottomMargin}"
+                      HorizontalAlignment="Left">
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <Grid>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="200"/>
+                                <ColumnDefinition Width="200"/>
+                            </Grid.ColumnDefinitions>
+                            <Border Background="{ThemeResource SystemBaseLowColor}"
```

Comment:
Updating mapping logic from only to 1:1 mapping to upto 3:3 keys mapping.

Reply:
Will update the mapping logic in a separate PR with adding settings schema and events.